lib/src/builder/mod.rs

```diff
@@ -76,22 +76,49 @@ where
 
     /// Initializes the database from the input.
     pub fn initialize_database<T: DbInitStrategy<D>>(self) -> Result<Self> {
-        T::initialize_database(self)
+        // Already have accounts u need from Host DB
+        // Form sparse MPT from DB -> bunch of *keccaks* 🌟
```

Comment:
Some extra comments here and also elsewhere in this file that I think are a bit too specific to track performance so can be removed